sites/partners/src/components/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -277,16 +318,52 @@ const BuildingDetails = ({
             />
           </Grid.Cell>
           <Grid.Cell className="seeds-grid-span-2">
-            <Field
-              label={t("listings.yearBuilt")}
-              name={"yearBuilt"}
-              id={"yearBuilt"}
-              placeholder={t("listings.yearBuilt")}
-              type={"number"}
-              register={register}
-            />
+            {enableRegions ? (
+              <FieldValue label={t("t.region")}>
+                <Select
+                  id="region"
+                  name="region"
+                  placeholder={t("listings.sections.regionPlaceholder")}
+                  register={register}
+                  inputProps={{
+                    className: "w-full",
+                    onChange: () => setValue("neighborhood", undefined),
+                  }}
+                  options={Object.keys(RegionEnum).map((entry) => ({
+                    value: entry,
+                    label: entry.toString().replace("_", " "),
+                  }))}
+                />
+              </FieldValue>
+            ) : (
+              <Field
+                label={t("listings.yearBuilt")}
+                name={"yearBuilt"}
+                id={"yearBuilt"}
+                placeholder={t("listings.yearBuilt")}
+                type={"number"}
+                register={register}
+              />
+            )}
           </Grid.Cell>
         </Grid.Row>
+        <GridRow columns={3}>
+          <GridCell>
+            <p className="field-sub-note">{t("listings.requiredToPublish")}</p>
```

Comment:
Issue: The "required to publish" is floating pretty far below the fields 

This PR:
![image](https://github.com/user-attachments/assets/e78ff676-cd57-49f7-b999-3737ca502caa)

In Main:
![image](https://github.com/user-attachments/assets/3f77dd19-58cf-4e6d-b89e-d00b7b44668e)